pkg/provider/azure.go

```diff
@@ -811,6 +813,10 @@ func (az *Cloud) initCaches() (err error) {
 		return err
 	}
 
+	getter := func(key string) (interface{}, error) { return nil, nil }
+	if az.StorageAccountCache, err = azcache.NewTimedCache(time.Minute, getter, false); err != nil {
```

Comment:
Storage account will be supported by ARG so could you pass `az.Config.DisableAPICallCache` instead of hard coding it as false?

Reply:
How about creating a `newStorageAccountCache()` method so it can be reused?

Reply:
fixed